src/assets/sass/protocol/components/_split.scss

```diff
@@ -196,9 +196,9 @@
             // grid is all start/end based we don't need bidi declarations for RTL support \o/
             align-items: center;
             display: grid;
-            grid-template-columns: 1fr 1fr;
-            grid-template-areas: "body media";
             grid-column-gap: get-theme('h-grid-lg');
+            grid-template-areas: "body media";
```

Comment:
curiousity: are these changes for alphabetic property re-ordering?

Reply:
Correct